scipy/stats/_continuous_distns.py

```diff
@@ -2900,6 +2900,135 @@ def _entropy(self, c):
 genhalflogistic = genhalflogistic_gen(a=0.0, name='genhalflogistic')
 
 
+class genhyperbolic_gen(rv_continuous):
+    r"""A generalized hyperbolic continuous random variable.
+
+    %(before_notes)s
+
+    Notes
+    -----
+    The probability density function for `genhyperbolic` is:
+
+    .. math::
+
+        f(x, \lambda, \alpha, \beta, \delta, \mu) =
+           \frac{(\gamma/\delta)^\lambda}{\sqrt{2\pi}K_\lambda(\delta \gamma)}
+           e^{\beta (x - \mu)} \times \frac{K_{\lambda - 1/2}
+           (\alpha \sqrt{\delta^2 + (x - \mu)^2})}
+           {(\sqrt{\delta^2 + (x - \mu)^2} / \alpha)^{1/2 - \lambda}}
+
+    for :math:`x \in (- \infty; \infty)`,
+    :math:`\lambda, \mu \in \mathbb{R}`,
+    :math: `0 \le |\beta| < \alpha ` \beta,
+    and :math: `\delta \in \mathbb{R}_{>0}`.
+
+    :math:`\gamma := \sqrt{\alpha^2 - \beta^2}`.
+    :math: `K_{\lambda}(.)` denotes the modified Bessel function of the third
+    kind and order :math: `\lambda` (`scipy.special.kn`)
```

Comment:
I'm not sure if it's required but can't hurt to remove the blank space after :math: on both of these lines.